migrations/versions/fd3c612364cf_add_benchalerts.py

```diff
@@ -0,0 +1,61 @@
+"""add benchalerts
+
+Revision ID: fd3c612364cf
+Revises: cfafc733f81d
+Create Date: 2023-06-12 13:04:56.343773
+
+"""
+import sqlalchemy as sa
+from alembic import op
+from sqlalchemy.dialects import postgresql
+
+# revision identifiers, used by Alembic.
+revision = "fd3c612364cf"
+down_revision = "cfafc733f81d"
+branch_labels = None
+depends_on = None
+
+
+def upgrade():
+    op.create_table(
+        "benchalerts_run",
+        sa.Column("id", sa.String(), nullable=False),
+        sa.Column("benchmarkable_id", sa.String(), nullable=False),
+        sa.Column("reason", sa.String(), nullable=False),
+        sa.Column("finished_at", sa.DateTime(), nullable=True),
+        sa.Column("output", postgresql.JSONB(astext_type=sa.Text()), nullable=True),
+        sa.Column("status", sa.String(), nullable=True),
+        sa.ForeignKeyConstraint(
+            ["benchmarkable_id"],
+            ["benchmarkable.id"],
+        ),
+        sa.PrimaryKeyConstraint("id"),
+    )
+    op.drop_table("notification")
```

Comment:
I think I won't spend time writing a migration that turns old pending notifications into benchalerts runs. What that means is that, depending on when this is merged, currently-running benchmarks might not post any notifications. I'll try not to merge when benchmarks are running to minimize this chance, but even if that does happen it should be fine.

Also we'll lose the records of past notifications, but I think that's also low-risk.